packages/v2/gui/src/sak/ung-meldinger/UngMessages.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import type {
+  InformasjonsbrevMottakerValgResponseIdType,
+  KodeverdiSomObjektDokumentMalTypeKilde,
+} from '@k9-sak-web/backend/ungsak/generated';
+import type { Meta, StoryObj } from '@storybook/react';
+import { useState } from 'react';
+import withMaxWidth from '../../storybook/decorators/withMaxWidth';
+import { FakeUngMessagesBackendApi } from '../../storybook/mocks/FakeUngMessagesBackendApi';
+import { UngMessages } from './UngMessages';
+import type { UngMessagesFormState } from './UngMessagesFormState';
+
+const meta: Meta<typeof UngMessages> = {
+  title: 'gui/sak/ung-meldinger/UngMessages',
+  decorators: [withMaxWidth(420)],
+  component: UngMessages,
+};
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  render: () => {
+    const [formValues, setFormValues] = useState<UngMessagesFormState | undefined>(undefined);
+    const mockApi = new FakeUngMessagesBackendApi();
+    const brevmaler = [
+      {
+        malType: {
+          kode: 'GENERELT_FRITEKSTBREV',
+          kodeverk: 'DOKUMENT_MAL_TYPE',
+          navn: 'Fritekst generelt brev',
+          kilde: 'GENERELT_FRITEKSTBREV' as KodeverdiSomObjektDokumentMalTypeKilde,
```

Comment:
Foreslår å bruke generert konstant for enum verdi istadenfor å caste string til enum type. Må då fjerne `type` frå import av `KodeverdiSomObjektDokumentMalTypeKilde`

```suggestion
kilde: KodeverdiSomObjektDokumentMalTypeKilde.GENERELT_FRITEKSTBREV,
```